modules/Lumasachi/app/Http/Controllers/OrderController.php

```
@@ -29,19 +29,21 @@
+            ->when($user->isEmployee(), function ($query) use ($user) {
+                $query->where('assigned_to', $user->id)
+                    ->orWhere('created_by', $user->id);
+            })
+            ->when($user->isAdministrator() || $user->isSuperAdministrator(), function ($query) use ($user) {
+                // Get all the orders
+                $query->get();
+            })
             ->get();
 
-        return response()->json(OrderResource::collection($orders));
+        return response()->json(OrderResource::collection($ordersQuery));
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Lumasachi\app\Http\Resources\OrderResource' in method 'index'. (Clean Code Rules)

(StaticAccess)